leveler/commands/lvladmin/settings.py

```diff
@@ -61,6 +62,13 @@ async def overview(self, ctx):
         )
         await ctx.send(embed=em)
 
+    @checks.is_owner()
+    @lvladmin.command()
+    async def resetrep(self, ctx):
+        """Resets all reputation points from MonogoDB (Bot Owner Only)"""
+        await self.db.users.update_many({}, {"$set": {"rep": 0}})
+        await ctx.send("**All reputation points have been removed.**")
```

Comment:
```suggestion
        async with ctx.typing():
            await self.db.users.update_many({}, {"$set": {"rep": 0}})
            await ctx.send("**All reputation points have been removed.**")
```

This can take some time on big databases, lets add an typing indicator to show thats command is really doing something during that time.